cargo-near/src/util/mod.rs

```diff
@@ -276,9 +276,28 @@ pub(crate) fn extract_abi_entries(
     unsafe {
         let lib = libloading::Library::new(dylib_path)?;
         for symbol in near_abi_symbols {
-            let entry: libloading::Symbol<fn() -> near_abi::__private::ChunkedAbiEntry> =
+            let entry: libloading::Symbol<extern "C" fn() -> *const std::ffi::c_char> =
                 lib.get(symbol.as_bytes())?;
-            entries.push(entry().clone());
+            match serde_json::from_slice(std::ffi::CString::from_raw(entry() as *mut _).to_bytes())
```

Comment:
We might want https://doc.rust-lang.org/std/slice/fn.from_raw_parts.html here instead of going through CString now

Reply:
Yeah, already patched in #80. I'll merge in here. Also, it would be `Vec::from_raw_parts` otherwise we'd be leaking memory.